packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -116,6 +133,71 @@ export interface BaseServiceProps extends BaseServiceOptions {
   readonly launchType: LaunchType;
 }
 
+/**
+ * Base class for configuring listener when registering targets.
+ */
+export abstract class ListenerConfig {
+  /**
+   * Create a config for adding target group to ALB listener.
+   */
+  public static applicationListenerConfig(listener: elbv2.ApplicationListener, props?: elbv2.AddApplicationTargetsProps): ListenerConfig {
```

Comment:
I would remove the `Config` at the end of the method name here. Shorter is better, and the "config" part is already implied by the class name.

Reply:
:+1: